tests/local_clean_urls.php

```diff
@@ -209,6 +209,21 @@ public function test_local_cleanurls_simple() {
         // If we change url config then we need to throw away the cache.
         purge_all_caches();
 
+        $url = 'http://www.example.com/moodle/mod/forum/user.php?id=' . $this->staff->id . '&mode=discussions';
```

Comment:
Can you also please test the case where the url params are the other way around eg ?mode=discussions&id=' . $this->staff->id;